test/Intl/DateTimeFormatOptions.js

```diff
@@ -51,7 +51,7 @@ var tests = [
                 }
                 catch (e) {
                     if (!(e instanceof RangeError)) {
-                        fail("Incorrect exception was thrown.");
+                        assert.fail("Incorrect exception was thrown.");
```

Comment:
>assert.fail("Incorrect exception was thrown."); [](start = 24, length = 47)

I'll add this change to my PR here: [https://github.com/Microsoft/ChakraCore/pull/2818](https://github.com/Microsoft/ChakraCore/pull/2818)